src/librustc/diagnostics.rs

```diff
@@ -642,7 +642,7 @@ attributes:
 #![no_std]
 ```
 
-See also https://doc.rust-lang.org/book/first-edition/no-stdlib.html
+See also https://doc.rust-lang.org/unstable-book/language-features/lang-items.html#writing-an-executable-without-stdlib
 "##,
```

Comment:
I just don't know how to wrap this line.

Reply:
You could reword it like 

```markdown
See also [«some description here»][1].

[1]: https://doc.rust-lang.org/unstable-book/language-features/lang-items.html#writing-an-executable-without-stdlib
```

Reply:
The link itself is longer than 100 characters, which is prohibited by the lint. :man_shrugging:

Reply:
@lzutao Tidy allows long lines written in the form `[stuff]: http://url`.

https://github.com/rust-lang/rust/blob/a54b5c7a645ead203d77e78245362f9e0f00dd3c/src/tools/tidy/src/style.rs#L47-L53